tools/resource-gen/genutils/util.go

```diff
@@ -33,7 +34,8 @@ func SelectorsForMessage(m protoreflect.MessageDescriptor) []string {
 		m := field.Message()
 		if m != nil && m.FullName() == "kuma.mesh.v1alpha1.Selector" {
 			fieldName := string(field.Name())
-			selectors = append(selectors, strings.Title(fieldName))
+			caser := cases.Title(language.English)
```

Comment:
Why are we hardcoding English? How did it work before? Why do we need to change it at all?

Reply:
`strings.Title` is deprecated. Which of course used English